src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,291 @@
+        String firstWord = input[0];
+        switch (firstWord) {
+        case "deadline":
+            addDeadline(input,length);
+            break;
+        case "event":
+            addEvent(input,length);
+            break;
+        case "todo":
+            addTodo(input,length);
+            break;
+        default:
+            throw new DukeException(Response.UNSPECIFIED_TASK);
+        }
+    }
+
+    private static void printList() {
+        System.out.println("Accessing archives...");
+        int i = 1;
+        for (Task num : commands) {
+            System.out.println(i + ". " + num);
+            i += 1;
+        }
+    }
+
+    private static void markDone(int doneTaskNumber) throws IOException {
+        (commands.get(doneTaskNumber)).markAsDone();
+        if (loadFlag == 1) {
+            System.out.println("The following task has been marked as done Master!");
+            System.out.println((doneTaskNumber + 1) + ". " + commands.get(doneTaskNumber));
+            saveAllTasks();
+        }
+    }
+
+    private static void deleteTask(int doneTaskNumber) throws IOException {
+        System.out.println("Taking one last look Master, at this Task. Removing the following from my memory");
+        System.out.println((doneTaskNumber+1) + ". " + commands.get(doneTaskNumber));
+        commands.remove(commands.get(doneTaskNumber));
+        positionCheck -= 1;
+        System.out.println("Goodbye Task, may the force be with you. You have " + positionCheck + " task(s) left Master");
+        saveAllTasks();
+    }
+
+    public static void greetUser() {
+        System.out.println(Response.GREETINGS);
+    }
+
+    public static void saveNewTask(String[] input) throws IOException {
+        String filePath = new File("Tasks.txt").getAbsolutePath();
+        FileWriter fw = new FileWriter(filePath, true);
+        String fullTaskAsString = "";
+        for (String individualString : input) {
+            fullTaskAsString += individualString + " ";
+        }
+        fullTaskAsString += "| 0";
+        fw.write(fullTaskAsString + "\n");
+        fw.close();
+    }
+
+    public static void saveAllTasks() throws IOException {
+        String filePath = new File("Tasks.txt").getAbsolutePath();
+        FileWriter fw = new FileWriter(filePath, false);
+        String taskInFile;
+        String done;
+        for (Task individualTask : commands) {
+            if (individualTask instanceof Deadline) {
+                taskInFile = "deadline " + individualTask.description + " /by " + ((Deadline) individualTask).by;
+            } else if (individualTask instanceof Event) {
+                taskInFile = "event " + individualTask.description + " /at " + ((Event) individualTask).at;
+            } else {
+                taskInFile = "todo " + individualTask.description;
+            }
+            if (individualTask.isDone) {
+                done = "1";
+            } else {
+                done = "0";
+            }
```

Comment:
Perhaps, use of conditional operator would be good as it will help reduce the number of lines 😃